developers/uploader/README.md

```diff
@@ -0,0 +1,61 @@
+# Uploader
+
+### What's Uploader?
+
+The Uploader represents a cutting-edge solution designed to streamline the storage management process within a decentralized network. Built with efficiency and scalability in mind, Uploader leverages advanced technologies to provide secure, reliable, and cost-effective storage solutions to users.
+
+### Architecture Overview
+
+Uploader is built on a robust architecture that seamlessly integrates various components to ensure optimal performance. The architecture consists of:
+
+- Uploader API Layer: Exposes both public and private APIs for frontend and microservices interactions, respectively.
+- 1-N Storage Microservices: Multiple microservices, each specializing in different storage types, responsible for handling storage operations.
+- IPFS Integration: Temporary storage management using the InterPlanetary File System (IPFS).
+
+### Features
+
+Uploader offers a range of powerful features tailored to meet the needs of decentralized storage management:
+
+- User Content Uploads: Users can seamlessly upload their content through the user-friendly frontend interface.
+- Payment Handling: Uploader integrates with payment systems to manage the financial aspects of storage services.
+- Decentralized Storage: Content is pushed to decentralized storage networks like Filecoin and Arweave for enhanced security and redundancy.
+- API Documentation: Comprehensive API documentation on each repo to allow users to understand and interact with the system effortlessly.
+- Uploader.js: a TypeScript library designed to simplify interaction with the Uploader API. This library provides a user-friendly and intuitive interface for calling API endpoints within the Uploader Storage system.
+
+### Components
+
+- [Uploader](https://github.com/oceanprotocol/decentralized_storage_backend)
+- [Uploader.js](https://github.com/oceanprotocol/uploader.js)
+- [Uploader UI](https://github.com/oceanprotocol/uploader-ui-lib)
+
+### Microservices:
+
+- [Filecoin](https://github.com/oceanprotocol/uploader_filecoin) (WIP)
+- [Arweave](https://github.com/oceanprotocol/uploader_arweave)
+
+### User Workflow
+
+Uploader simplifies the user workflow, allowing for easy management of storage operations:
+
+- Users fetch available storage types and payment options from the frontend.
+- Quotes for storing files on the Microservice network.
+Files are uploaded from the frontend to Uploader, which handles temporary storage via IPFS.
+- The Microservice takes over, ensuring data is stored on the selected network securely.
+- Users can monitor upload status and retrieve links to access their stored content.
+
+<figure><img src="../../.gitbook/assets/uploader/uploader_1.png" alt=""><figcaption><p>Ocean Uploader - storage flow 1</p></figcaption></figure>
```

Comment:
Both of the diagrams mention DBS, we need to update the naming. I suggest we describe some of the steps that are covered in the diagrams.

Reply:
who do we have to ask for these diagrams @AnaLoznianu ?

Reply:
I asked the designer. We'll receive them by Monday